modules/software-extensibility/pages/bonita-repository-access.adoc

```diff
@@ -0,0 +1,149 @@
+    <!-- ...  -->
+    <profiles>
+        <profile>
+            <id>bonitasoft</id>
+            <repositories>
+                <repository>
+                    <id>bonitasoft-releases</id>
+                    <name>Bonitasoft Releases</name>
+                    <releases>
+                        <enabled>true</enabled>
+                    </releases>
+                    <snapshots>
+                        <enabled>false</enabled>
+                    </snapshots>
+                    <url>https://bonitasoft.jfrog.io/artifactory/maven</url>
+                </repository>
+            </repositories>
+            <pluginRepositories>
+                <pluginRepository>
+                    <id>bonitasoft-releases</id>
+                    <name>Bonitasoft Releases</name>
+                    <releases>
+                        <enabled>true</enabled>
+                    </releases>
+                    <snapshots>
+                        <enabled>false</enabled>
+                    </snapshots>
+                    <url>https://bonitasoft.jfrog.io/artifactory/maven</url>
+                </pluginRepository>
+            </pluginRepositories>
+        </profile>
+    </profiles>
+    <!-- ...  -->
+  <activeProfiles>
+    <activeProfile>bonitasoft</activeProfile>
+  </activeProfiles>
+
+</settings>
+----
+
+[IMPORTANT]
+====
+Do not forget to mark bonitasoft profile as `active` in the `activeProfiles` section or the declared repositories will not be used for dependency resolution.
+====
+
+
+=== Authentication
+
+Now that maven is aware of Bonita repositories, we have to give it the credentials to be allowed to download artifacts. Here is the simple way to specify credentials to use with https://maven.apache.org/settings.html#Servers[repository servers] but we strongly advice users to use https://maven.apache.org/guides/mini/guide-encryption.html[password encryption] to enforce security.
+
+[source, xml]
+.Server authentication in settings.xml
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 https://maven.apache.org/xsd/settings-1.0.0.xsd">
+
+    <!-- ...  -->
+    <servers>
+        <server>
+            <!-- This server ID must be the same as the one use for repository declaration ! -->
+            <id>bonitasoft-releases</id>
+            <username>MY_LOGIN</username>
+            <password>MY_BONITA_REPOSITORY_TOKEN</password>
+        </server>
+    </servers>
+    <!-- ...  -->
+</settings>
+----
+
+=== Maven configuration in Bonita Studio
+
+If you want to set up access to Bonita Artifact Repository using Bonita Studio, please refer to the dedicated page on how to xref:setup-dev-environment:configure-maven.adoc[configure Maven from Bonita Studio].
+
+== Docker
```

Comment:
Warning: we are not supposed to deliver docker image in the repository for now on. This will create confusion

Reply:
Ok I remove this section